src/translations/en.json

```diff
@@ -9236,7 +9236,7 @@
               "documentation": "[%key:ui::panel::config::logs::detail::documentation%]",
               "issues": "[%key:ui::panel::config::logs::detail::issues%]",
               "first_occurred": "[%key:ui::panel::config::logs::detail::first_occurred%]",
-              "occurrences": "[%key:ui::panel::config::logs::detail::occurrences%]",
+              "number_of_occurrences": "[%key:ui::panel::config::logs::detail::number_of_occurrences%]",
```

Comment:
Looks like this one is not used, right? I can not find the reference in the codebase